app/_locales/en/messages.json

```diff
@@ -1518,7 +1518,7 @@
   },
   "ledgerLive": {
     "message": "Ledger Live",
-    "description": "The name of a desktop app that can be used with your ledger device. We can also use it to connect a users Ledger device to MetaMask."
+    "description": "The name of a desktop app that can be used with your ledger device. We can also use it to connect an user's Ledger device to MetaMask."
```

Comment:
Hmm, looks like there's still a typo here. This should be "a user's".

Reply:
Nope, "an" is used before a word that starts with a vowel :) https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html

Reply:
Noted. Will change.